runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsOverlay.java

```diff
@@ -215,24 +278,62 @@ else if (poisonState > 0)
 			}
 
 			renderHealingBar(g, offsetHealthX, offsetHealthY,
-				maxHealth, currentHealth, height,
-				foodHealValue, HEAL_COLOR);
+					maxHealth, currentHealth, height,
+					foodHealValue, HEAL_COLOR);
 
 			renderHealingBar(g, offsetPrayerX, offsetPrayerY,
-				maxPrayer, currentPrayer, height,
-				prayerHealValue, PRAYER_HEAL_COLOR);
+					maxPrayer, currentPrayer, height,
+					prayerHealValue, PRAYER_HEAL_COLOR);
+
+			if (resizedBottomMode)
+			{
+				renderHealingBar(g, offsetEnergyX, offsetEnergyY,
+						maxEnergy,
+						currentEnergy,
+						height,
+						energyHealValue,
+						ENERGY_HEAL_COLOR
+				);
+			}
 		}
 
 		if (config.enableSkillIcon() || config.enableCounter())
 		{
 			final BufferedImage healthImage = skillIconManager.getSkillImage(Skill.HITPOINTS, true);
 			final int counterHealth = client.getBoostedSkillLevel(Skill.HITPOINTS);
 			final int counterPrayer = client.getBoostedSkillLevel(Skill.PRAYER);
+			final int counterEnergy = client.getEnergy();
+			final int counterWeight = client.getWeight();
```

Comment:
Not a big fan of this naming, since counterweight is something else, but i guess it follows suite.

Reply:
well, feel free to suggest something...

counterInventoryWeight?
counterMass?